docker/Dockerfile.eurekamirror

```diff
@@ -0,0 +1,3 @@
+FROM scratch
```

Comment:
Add a comment stating that this container is used only for internal testing and what it does.